package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@moxy/express-ensure-content-type",
+  "version": "0.0.0",
+  "description": "Express middleware that ensures requests match the specified content-type",
+  "author": "André Cruz <andre@moxy.studio>",
+  "license": "MIT",
+  "homepage": "https://github.com/moxystudio/express-ensure-content-type#readme",
+  "repository": "github:moxystudio/express-ensure-content-type",
+  "bugs": "https://github.com/moxystudio/express-ensure-content-type/issues",
+  "keywords": [
+    "express",
+    "content-type",
+    "media-type",
+    "enforce",
```

Comment:
add `"ensure"` as a synonim to `"enforce"`

Reply:
Done.